Makefile

```
@@ -282,11 +289,23 @@ install-bash-completion:
 GIT_VERSION: force
 	@if [ "$(GIT_VERSION)" != "`cat 2>/dev/null GIT_VERSION`" ] ; then echo "$(GIT_VERSION)" >GIT_VERSION; fi
 
+build-deb: ## Build deb package of cilium.
+	$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C ./contrib/packaging/deb
+
+build-rpm: ## Build rpm package of cilium.
+	$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C ./contrib/packaging/rpm
+
+runtime-tests: ## Run runtime-tests for Cilium.
+	$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C tests runtime-tests
+
+k8s-tests:
+	$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C tests k8s-tests
+
 -include Makefile.docker
 
+##@ API targets
 CRD_OPTIONS ?= "crd:crdVersions=v1"
-# Generate manifests e.g. CRD, RBAC etc.
-manifests:
+manifests: ## Generate manifests e.g. CRD, RBAC etc.
```

Now that we are labeling this "officially", let's make it be slightly more specific:

```suggestion
manifests: ## Generate K8s manifests e.g. CRD, RBAC etc.
```